### PR TITLE
Add support for MLPERF optimized pipeline from example

### PR DESCRIPTION
**Pull request by ANSHUMAN87**
Add support for MLPERF SD XL pipeline

### CODE REVIEW

**Comment by ANSHUMAN87** [comment]
@regisss Gentle reminder! To merge the PR. Please let me know if anything needs to be done from my end. Thanks!

JH...!

File: optimum/habana/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_mlperf.py [hunk 1]
```diff
@@ -170,6 +170,7 @@ def __init__(
         )
         self.unet.set_default_attn_processor = set_default_attn_processor_hpu
         self.unet.forward = gaudi_unet_2d_condition_model_forward
+        self.quantized = False
```
Comment (yeonsily) [inline optimum/habana/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_mlperf.py:173]: why quantized false here? is this working only for bf16?

**Comment by yeonsily** [comment]
@ANSHUMAN87 Can you please add your case for diffuser CI also? https://github.com/huggingface/optimum-habana/blob/main/tests/test_diffusers.py

**Comment by kumarans-ai** [comment]
> @ANSHUMAN87 Can you please add your case for diffuser CI also? https://github.com/huggingface/optimum-habana/blob/main/tests/test_diffusers.py

@yeonsily  This is the base patch , there other PR's which will cover the test case part .

**Review by yeonsily — APPROVED** [review]

**Review by regisss — APPROVED** [review]

**Pull request merged**